samples/Audio/SimpleAudio/SimpleAudio.Game/SimpleAudio.Game.csproj

```diff
@@ -4,10 +4,10 @@
     <RootNamespace>SimpleAudio</RootNamespace>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Stride.Core" Version="4.1.0.1-beta" PrivateAssets="contentfiles;analyzers" />
```

Comment:
I believe this change is not what you're intending. When building Stride locally it will produce NuGet packages with the beta suffix. So the previous setup would have been correct.

Reply:
You mean the version specified in [this file](https://github.com/stride3d/stride/blob/master/sources/editor/Stride.Samples.Templates/ThisPackageVersion.DevBuild.cs)?
For me, even if I have not modified that file, the resulting packages in `NuGetDev` are like this:

![image](https://github.com/stride3d/stride/assets/8967302/a49e287e-cfdb-4621-8934-07c876d16424)

(I have upped the version to v4.2.0.1 in `SharedAssemblyInfo.cs` while testing)

As you see, at least for me, it has no `-beta` nor `-dev` suffixes.

Reply:
Indeed - seems like beta suffix was removed in this commit https://github.com/stride3d/stride/commit/7f383921a60f76ce200c3d4156a78f338568acba